docs/modules/ROOT/pages/reference/database.adoc

```diff
@@ -42,13 +42,13 @@ This document lists and details the database schema used by KubeArchive.
 |varchar not null
 |Version of the resource (`metadata.resourceVersion`).
 
-|created_ts
+|created_at
 |timestamp not null
-|Time when KubeArchive received the resource for the first time.
+|Timestamp when the record is inserted in this table
```

Comment:
For consistency, period at the end of the description.

```suggestion
|Timestamp when the record is inserted in this table.
```

Reply:
Fixed!